dsfs/package_test.go

```diff
@@ -31,9 +31,9 @@ func TestPackageFilepath(t *testing.T) {
 		{ipfs, "/ipfs/QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M/dataset.json", PackageFileMeta, "/ipfs/QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M/meta.json"},
 		{ipfs, "QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M", PackageFileDataset, "/ipfs/QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M/dataset.json"},
 
-		{mem, "/mem/QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M", PackageFileDataset, "/mem/QmZfwmhbcgSDGqGaoMMYx8jxBGauZw75zPjnZAyfwPso7M"},
```

Comment:
@b5 
I wasn't sure if these tests were here because we want test repos to return a filepath without a `/dataset.json` extension was because we didn't have a generalized `PackageFileDataset`, like it says in the TODO on line 81 of dsfs/package.go, or because we wanted it to behave this way.

Reply:
nice fix. I think this is great and should go in as-is